src/components/trendingBooks/TrendingBooks.tsx

```diff
@@ -0,0 +1,35 @@
+import BookCard from "./BookCard";
+
+const TRENDING_URL = "http://openlibrary.org/trending/daily.json?page=1";
+
+export default async function TrendingBooks() {
+  const trendingData = await fetch(TRENDING_URL);
+  const trendingBooks = await trendingData.json();
+  const {works} = trendingBooks;
```

Comment:
We should type these as best we can

Reply:
Also see my proposal below about moving fetch logic out of the component to a 'client' file?